modules/ssh/ssh.go

```diff
@@ -124,11 +124,11 @@ func sessionHandler(session ssh.Session) {
 	// Wait for the command to exit and log any errors we get
 	err = cmd.Wait()
 	if err != nil {
-		log.Error("SSH: Wait: %v", err)
+		log.Error("SSH: Wait: %v Failed authentication attempt from %s", err, session.RemoteAddr())
```

Comment:
Are you sure that every error from cmd wait will be due to an authentication failure?

Reply:
Nope, maybe a check if err is `exit status 1` but then err could also be other things and still be an authentication failure and I don't want to miss those. I'm not too familiar with the code base yet and only started using gitea this week. 

Another option is prepend it with 'Possible' or omit 'Failed authentication attempt' entirely, leaving just the IP and leave it to the user to decide whether to include this in fail2ban.